### ISSUE TITLE
twisted.conch.ssh.connection should be ported to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8660]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)| @einarfd reported|
|-|-|
|Trac ID|trac#8660|
|Type|enhancement|
|Created|2016-07-23 08:39:28Z|

A merge request for this bug can be found at https://github.com/twisted/twisted/pull/398

<details><summary>Searchable metadata</summary>

```
trac-id__8660 8660
type__enhancement enhancement
reporter__einarfd einarfd
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469263168613744 1469263168613744
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)|@einarfd removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 308036106372c1527458ea9d01c725e204d3335e
```
#!CommitTicketReference repository="" revision="308036106372c1527458ea9d01c725e204d3335e"
Merge pull request #400 from twisted/8660-einarfd-ssh-connection

Author: einarfd
Reviewer: rodrigc
Fixes: #8660

Port twisted.conch.ssh.connection to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
